CHANGELOG.md

```diff
@@ -9,6 +9,26 @@ Our versioning strategy is as follows:
 - Minor: may include breaking changes in framework packages (e.g. framework upgrades, new features, improvements)
 - Major: may include breaking changes in core packages (e.g. major architectural changes, major features)
 
+## 21.8.0
+
+### 🛠 Breaking Changes
+
+`[sitecore-jss-angular]` `[sitecore-jss-schematics]` `[templates/angular]` Upgrade Angular to v17 ([#1939](https://github.com/Sitecore/jss/pull/1939))
+  - Updated Angular and core dependencies to ~17.3.11
+  - Updated Typescript to ~5.2.2
+  - Updated import statements from zone.js/dist/zone-node to zone.js
+      Use _'zone.js'_ instead of _'zone.js/dist/zone-node'_
```

Comment:
IMO it would be cleaner if you put these instructions into docs/upgrades/../21.8/md and have a link to it here in release notes. Something like "You can find the uprade instruction in [here]".
Please also add the link to angular's v16->v17 upgrade guide for extra reference in our upgrade guide.

Reply:
@art-alexeyenko We can't really put these instructions under 21.8.md file, since we expect to publish migration guide to official doc side

Reply:
@illiakovalenko does it make sense to publish a two-point guide to doc site however? Our general approach is to have upgrade instructions available through github - in this case they will be.

Reply:
@art-alexeyenko Let's discuss more later, I will raise a question regarding all "migration" related files

Reply:
DIscussed this more.
Let's remove the instructions from github and post them on doc site as usual.